### ISSUE TITLE
Missing i18n in /p/api/index.php

### ISSUE THREAD

**Issue by math-GH** [FreshRSS/FreshRSS#6222]
This page is still in plain English and needs a translation

https://github.com/FreshRSS/FreshRSS/blob/b5445e1e56497f8bdc1de35a8325418c44b96d37/p/api/index.php#L25-L45

**Comment by aledeg** [comment]
I've translated this page but I am not sure if it is useful. So far, I only succeeded to display it in English. I am probably missing something though.

**Comment by math-GH** [comment]
> I've translated this page but I am not sure if it is useful. So far, I only succeeded to display it in English. I am probably missing something though.

Let's discuss it in the PR over there ;)
